machine/machine.go

```diff
@@ -313,3 +313,57 @@ func getNumCores() int {
 
 	return numCPU
 }
+
+var cpuToSocket map[int]int
+
+// GetSocketFromCPU returns socket which is bounded to passed CPU.
+// If there is no information it would return -1.
+func GetSocketFromCPU(cpu int) int {
```

Comment:
This function retives infromation from sysfs so it fits better to [sysinfo](https://github.com/google/cadvisor/tree/master/utils/sysinfo) or [sys](https://github.com/google/cadvisor/tree/master/utils/sysfs), please move it if it is possible.

Reply:
Functionality moved to ``utils/sysinfo/GetSocketFromCPU``.